### PR TITLE
cleans up bone hurting juice messages

### PR DESCRIPTION
**Pull request by ghost**
mildly annoying

### CODE REVIEW

**Comment by Barhandar** [comment]
Restore missing muscles part, it only fires for skeletons who don't have flesh.

**Comment by McBawbaggings** [comment]
should have read the code before removing that part

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -939,10 +939,10 @@
 		else //you just want to socialize
 			if(bp)
 				playsound(M, get_sfx("desceration"), 50, TRUE, -1)
-				M.visible_message("<span class='warning'>[M] rattles loudly and flails around!!</span>", "<span class='danger'>Your bones so much your missing muscles spasm!!</span>")
+				M.visible_message("<span class='warning'>[M] rattles loudly and flails around!!</span>", "<span class='danger'>Your bones hurt so much that your missing muscles spasm!!</span>")
 				M.say("OOF!!", forced="bonehurtingjuice")
 				bp.receive_damage(200, 0, 0) //But I don't think we should
 			else
-				to_chat(M, "<span class='warning'>Your missing arm aches from whever you left it.</span>")
+				to_chat(M, "<span class='warning'>Your missing arm aches from wherever you left it.</span>")
```
Comment (McBawbaggings) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: was very tempted to try and shoe horn the phrase "phantom pain" but i couldn't think of a good way to convey it

**Review by SpaceManiac — APPROVED** [review]

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm (hunk 1 above)
Comment (MrDoomBringer) [reply code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: "You feel a phantom pain from your missing arm"?

**Pull request merged**